src/api/reflector.rs

```diff
@@ -207,7 +207,7 @@ where
             .client
             .request_events::<WatchEvent<K>>(req)
             .await?
-            .boxed_local(); // We use boxed_local to remove the Send requirement as we're not shipping this between threads
+            .boxed(); // We use boxed_local to remove the Send requirement as we're not shipping this between threads
```

Comment:
Shouldn't the comment be fixed too? 🤔

Reply:
Whoops, that was fixed in 403c36330022b32982dce074513329ec1f4ad0cd. Sorry for the noise.